app.py

```diff
@@ -1,6 +1,15 @@
 from flask import Flask
+import random
```

Comment:
In order to know which packages are from the standard library, your project or third party, you can separate them logically on the top of your file. This is well defined in the python style guide ([PEP8](https://www.python.org/dev/peps/pep-0008/#imports)).

![image](https://user-images.githubusercontent.com/9048523/62222276-f2526400-b367-11e9-9eda-21261ad1a051.png)

Meaning your imports would look like

```python
import random

from flask import Flask
```

The biggest benefit of this is, if you do it consistently, anyone who reads these packages will know which ones are which,

Reply:
There is also a tool that can do this for you, called [isort](https://github.com/timothycrosley/isort).

Reply:
There are also some minor formatting inconsistencies in your file with PEP8, if you want you can use an autoformatter for that as well [autopep8](https://github.com/hhatto/autopep8) - both this and isort are just small CLIs that allow you to format your code easily in a consistent way. 

[black](https://github.com/psf/black) is also a popular alternative to `autopep8`, I tend to prefer `black` but that is just a personal preference :)